src/app/pages/organizations/developers/developers-view.jsx

```diff
@@ -20,7 +20,7 @@ import {
   ChplFilterSearchBar,
   useFilterContext,
 } from 'components/filter';
-import { ChplLink, ChplPagination, ChplSortableHeaders } from 'components/util';
+import { ChplLink, ChplPagination, ChplLoadingTable,  ChplSortableHeaders } from 'components/util';
```

Comment:
```suggestion
import { ChplLink, ChplPagination, ChplLoadingTable, ChplSortableHeaders } from 'components/util';
```

You know how you see slight differences in 4px vs 8px rounded edges? I see similar things in spacing... 😁